### DISCUSSION TITLE
Add RTTY

### DISCUSSION THREAD

**Discussion by Adivor** [foldynl/QLog#298] [Ideas]
Thanks for this great job!!!!

As for the excellent cw, it wouldn't be bad to also have the possibility of rtty and cw full extended to icom rig radio.

**Comment by foldynl** [comment]
I'll give you a secret trick. If you have installed FLDigi , then you can define CW Key where Model is FLDigi in QLog. And then via QLog's CW console, you can send text to FLDigi and FLDigi will send a "correct sound" to your rig - RTTY, PSK or other digi modes supported by FLDigi. I used this trick with my Icom7300.

**Comment by Adivor** [comment]
mmmmm....good trick, i will try

THANKS
